README.md

```diff
@@ -18,12 +18,6 @@
 
 * Flutter: [FlutterFire plugins](https://github.com/flutter/plugins/blob/master/FlutterFire.md)
 
-* Node (via dart2js): [Anatoly Pulyaevskiy](https://github.com/pulyaevskiy) has
-  been working on unofficial wrappers.
-  * [package:firebase_admin_interop](https://pub.dev/packages/packages/firebase_admin_interop)
```

Comment:
Would you fix them instead?

https://pub.dev/packages/firebase_admin_interop etc etc

Reply:
Done! But I don't see an interop package for cloud firestore: https://pub.dev/packages?q=firestore_interop

Reply:
I found the repository, but maybe the package was taken down https://github.com/pulyaevskiy/firestore-interop